cmd/skopeo/signing.go

```diff
@@ -88,3 +89,137 @@ var standaloneVerifyCmd = cli.Command{
 	ArgsUsage: "MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT SIGNATURE",
 	Action:    standaloneVerify,
 }
+
+func addSignature(context *cli.Context) error {
+	if len(context.Args()) != 2 {
+		return errors.New("Usage: skopeo add-signature reference key-fingerprint")
+	}
+	ref := context.Args()[0]
+	fingerprint := context.Args()[1]
+
+	imageRef, err := transports.ParseImageName(ref)
+	if err != nil {
+		return fmt.Errorf("Error parsing %s: %v", ref, err)
+	}
+	name := transports.ImageName(imageRef)
```

Comment:
Why not use `ref` (the original user input) directly? Using exactly the users' input is simpler, and could make it less confusing.